keyboards/wilba_tech/wt75_c/info.json

```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "wilba.tech WT75-A", 
+    "url": "https://wilba.tech", 
+    "maintainer": "Wilba", 
+    "bootloader": "atmel-dfu", 
```

Comment:
The `bootloader` key is actually deprecated from the `info.json` spec, because QMK API reads this from the `rules.mk` file now.

Not hurting anything to my knowledge if you want to leave it in. Just an FYI.

Reply:
thanks